eisenhower-matrix/src/main/webapp/script.js

```diff
@@ -41,6 +41,20 @@ function loginStatus(status) {
   }
 }
 
+// checks if task if empty or only includes whitespace
+function checkTitle () {
+    let title = document.getElementById("taskName");
+    title.value = title.value.replace(/^\s+/, '').replace(/\s+$/, '');
+
+    if (title.value === '') {
+        /* title is empty or only whitespace */
+        document.getElementById('taskName').style.borderColor = "red";
+    } else {
+        /* title is valid */
+        getTasks();
```

Comment:
this getTasks() is already running each time the page reloads so it might be better to just return true or false like the tutorial linked suggests :)